packages/sanity/src/_internal/cli/actions/schema/utils/schemaApiClient.ts

```diff
@@ -0,0 +1,19 @@
+import {type CliApiClient} from '@sanity/cli'
+
+export function createSchemaApiClient(apiClient: CliApiClient) {
+  const client = apiClient({
+    requireUser: true,
+    requireProject: true,
+  }).withConfig({apiVersion: 'v2025-03-01', useCdn: false})
+
+  const projectId = client.config().projectId
+  const dataset = client.config().dataset
+  if (!projectId) throw new Error('Project ID is not defined')
```

Comment:
Won't requireUser + requireProject throw if either is missing? Seems fine to use a non-null assertion on both of these